test/subscription.js

```diff
@@ -598,43 +598,6 @@ test('subscription server sends update to subscriptions with custom context', t
   })
 })
 
-test('subscription server register handle function arg is not empty', t => {
-  const app = Fastify()
-  const schema = `
-    type Query {
-      add(x: Int, y: Int): Int
-    }
-  `
-
-  const resolvers = {
-    add: async ({ x, y }) => x + y
-  }
-
-  t.tearDown(app.close)
-
-  app.register(GQL, {
-    schema,
-    resolvers,
-    subscription: true
-  })
-
-  app.listen(0, err => {
-    t.error(err)
-
-    const url = 'ws://localhost:' + (app.server.address()).port + '/'
```

Comment:
This test was testing what happens if you connect to the root route with a websocket connection and expecting a not found response. I imagine this was mostly here for 100% test coverage, but I think that registering mercurius probably shouldn't have any bearing on the routes outside mercurius' context, so I just removed this test. `fastify-websocket` no longer supports the `handle` function during registration that creates the not found handler I believe.